english/knife.md

```diff
@@ -0,0 +1,6 @@
+You stab this miterious creature over and over 
+until your knife falls down, and so does the creature...
+
+[Look at the fallen creature's face](../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.